.circleci/config.yml

```diff
@@ -231,6 +231,14 @@ jobs:
       - go/install-gotestsum:
           gobin: $HOME/.local/bin
           version: 0.5.2
+      - run:
+          name: install statediff globally
+          command: |
+            mkdir -p /tmp/statediff
+            git clone https://github.com/filecoin-project/statediff.git /tmp/statediff
+            cd /tmp/statediff
+            go generate ./...
+            go install ./cmd/statediff
```

Comment:
We now run statediff as a binary. This has two advantages:

1. allows us to drop the go dependency, and thus mitigate the risk of dependency contagion (e.g. if statediff uses a newer version of go-state-types than Lotus, minimal version selection would've inadvertently chosen the newer one 😨 ).
2. we use statediff just like other non-Go implementations would use it, which makes us even better as a reference implementation ;-)

Reply:
https://github.com/filecoin-project/statediff/pull/15 will allow removal of the `go generate`